README.md

```diff
@@ -126,14 +126,22 @@ certificate-profile-name: my-profile-name
 ```
 
 ### File Specification
+#### Files List
+This strategy allows you to specify a comma separated list of files to be signed.
+
+```yaml
+# A comma separated list of absolute paths to the files being signed. Can be combined with the files-folder and file-catalog inputs.
+files: ${{ github.workspace }}\files\app.dll,${{ github.workspace }}\files\app.exe
```

Comment:
A more idiomatic pattern would be to use yaml block/flow sequences

```yaml
files:
- A
- B
- C
```

```yaml
files: [A, B, C]
```

Reply:
@riverar I'm not sure it's possible to use that syntax with a GitHub Action. Have you ever seen that before? I'm having trouble finding any more info with Google.
![image](https://github.com/Azure/trusted-signing-action/assets/59936622/3fcee2b6-db48-4264-afd7-8f7d52adcdf2)

Reply:
Hm. Some similar examples I dug up:

```yaml
on:
  label:
    types: [created, edited]
```
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onevent_nametypes

```yaml
steps:
  - name: My first action
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      FIRST_NAME: Mona
      LAST_NAME: Octocat
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-jobsjob_idstepsenv

Reply:
Interestingly GitHub's own cache action uses a string too. Maybe a GitHub Action parameter limitation?

```yaml
...
uses: actions/cache/restore@v4
with:
  path: |
    path/to/dependencies
    some/other/dependencies
```
https://github.com/actions/cache?tab=readme-ov-file#using-a-combination-of-restore-and-save-actions

Reply:
Looks like there's a long line of folks asking for this, hah! (https://github.com/orgs/community/discussions/11692). Sorry to lead you astray!

Perhaps you can tweak your PR to adapt the GitHub Cache-like pattern above? (Split on the string line breaks as opposed to a comma?)